valhalla/thor/matrix_common.h

```diff
@@ -56,36 +56,6 @@ struct Destination {
   }
 };
 
-// Will return a destination's date_time string
-inline std::string get_date_time(const std::string& origin_dt,
-                                 const uint64_t& origin_tz,
-                                 const baldr::GraphId& pred_id,
-                                 baldr::GraphReader& reader,
-                                 const uint64_t& offset) {
-  if (origin_dt.empty()) {
-    return "";
-  } else if (!offset) {
-    return origin_dt;
-  }
-  graph_tile_ptr tile = nullptr;
-  uint32_t dest_tz = 0;
-  if (pred_id.Is_Valid()) {
-    // get the timezone of the output location
-    auto out_nodes = reader.GetDirectedEdgeNodes(pred_id, tile);
-    dest_tz = reader.GetTimezone(out_nodes.first, tile) || reader.GetTimezone(out_nodes.second, tile);
```

Comment:
I just realized, this PR will also fix #4360 . this line is not what python-eyes think it is.. `dest_tz` will be true or false, i.e. 1 or 0. And 1 must be UTC. The new logic doesn't have that flaw.